account_move_line_group_by_extend/account_move_line.xml

```diff
@@ -6,7 +6,7 @@
             <field name="model">account.move.line</field>
             <field name="inherit_id" ref="account.view_account_move_line_filter"/>
             <field name="arch" type="xml">
-                <xpath expr="//group[@string='Group By...']/filter[@string='Period']" position="after">
+                <xpath expr="//group[@string='Group By']/filter[@string='Period']" position="after">
```

Comment:
Karen, for V9.0 attribute string can not be used due to translation issues. please @moylop260 trigger a new task to validte that also.

On thi cas I will merge just because  batch process (and in v8.0 that's not wrong).

regards.